content/en/logs/log_collection/javascript.md

```diff
@@ -160,6 +163,75 @@ This gives the following result:
 }  
 ```
 
+## Advanced usage
+
+### Filter by severity
+
+In some cases, you might want to disable the debug mode or to only collect warning and errors. This can be achieved by changing the logging level thanks to the `logLevel` parameter to `debug`, `info`, `warn`, and `error` :
+
+```
+Datadog.logger.setLogLevel('<SEVERITY_LEVEL>')
+```
+
+Only logs with a severity equal or higher to the specified one are sent.
+
+### Change the destination
+
+By default, the loggers are sending logs to Datadog. It is also possible to configure the logger to send logs to the console or to not send logs at all. This can be used in development environment to keep the logs locally.
```

Comment:
```suggestion
By default, the loggers are sending logs to Datadog. It is also possible to configure the logger to send logs to the console or not to send logs at all. This can be used in the development environment to keep the logs locally.
```